util/Migrator/SqlTableJournalExtensions.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.Data;
+using DbUp.Builder;
+using DbUp.Engine;
+using DbUp.Engine.Output;
+using DbUp.Engine.Transactions;
+using DbUp.SqlServer;
+
+namespace Bit.Migrator;
+
+public static class SqlTableJournalExtensions
```

Comment:
❓ Did you ever look into my comment about using `NullJournal`? Now that the implementation is here, what is the purpose of tracking scripts that you always want to run? If you have a folder for some that can always run, that's a use case for a different journal otherwise you'd keep using the one already in place. What am I missing in the requirements?